pelicun/assessment.py

```diff
@@ -651,45 +640,47 @@ def calculate_demand(
 
             if 'Units' in raw_demands.index:
                 raw_units = raw_demands.loc['Units', :]
-                raw_demands.drop('Units', axis=0, inplace=True)
+                raw_demands = raw_demands.drop('Units', axis=0)
 
             else:
                 raw_units = None
 
-            DEM_to_drop = np.full(raw_demands.shape[0], False)
+            dem_to_drop = np.full(raw_demands.shape[0], fill_value=False)
```

Comment:
[PEP 8 – Style Guide for Python Code: Function and Variable Names](https://peps.python.org/pep-0008/#function-and-variable-names).

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
> 
> Variable names follow the same convention as function names.

In my opinion we should be trying to use whole words separated by underscores.
However, we can revert the variable rename operations and disable the naming rules in our configuration.

Reply:
Okay, I did my homework, and I am convinced that we should adopt this rule. Using whole words instead of acronyms is an important part of the suggested approach that makes much more sense to me than lower case acronyms. In some cases, this will be easy to fix, but in others, say RID = residual interstory drift, I do not immediately see replacing the acronym with the whole words as a clear improvement. In any case, let's adopt the lower case rule now and work towards replacing those acronyms gradually.

Reply:
I'm happy you agree. In most cases it is possible to use a shorter set of whole words without compromising interpretability. E.g., `RID` could be `residual_drift`, or even `residual` if inside a function that deals with drifts, and so on.